src/Parse/ParseUser.php

```diff
@@ -192,9 +192,10 @@ public static function logInWithTwitter(
         $screen_name,
         $consumer_key,
         $consumer_secret = null,
-        $auth_token,
-        $auth_token_secret)
-    {
+        $auth_token, //@codingStandardsIgnoreLine
```

Comment:
This is fine for an ignore, but if you can put it between like so
```php
$consumer_secret = null,
//@codingStandardsIgnoreLine
$auth_token,
```
it should still work and won't be trailing after `$auth_token` instead.